src/persist-client/README.md

```diff
@@ -85,3 +85,34 @@ cargo bench -p mz-persist-client --bench=benches
 Larger writes are expected to take the above latency floor plus however long it
 takes to write the extra data to e.g s3. TODO: Get real numbers for larger
 batches.
+
+## Tracing
+
+Persist offers introspection into performance and behavior through integration
+with the [tracing] crate.
+
+[tracing]: https://docs.rs/tracing
+
+There are many policies we could adopt for what is instrumented and at what
+level. At the same time, it's early enough that there are lots of unknowns
+around what will be most useful for debugging real problems and what knobs are
+wanted to opt into additional detail. As a result, we adopt a common idiom of
+tracing at the persist API boundary with the rest of mz (as well as persist's
+API boundary with external systems such as s3 and aurora, which we wouldn't do
+if they themselves offered tracing integration). Concretely:
+
+- All persist spans have a `target` of `persist`, regardless of which crate or
```

Comment:
I don't think we should change the default behavior of the `target`, which is the module path, why not just require a new field called `area` or `context` that is `== "persist"`?

Reply:
I think I feel differently, but since we don't really have much information yet about how this "fine grained opt-in" will work in practice, I'm happy to punt this conversation and stick with the default for now. Removed!